src/main/java/seedu/address/logic/commands/AddContactToTeamCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.contact.Contact;
+import seedu.address.model.team.Team;
+
+import static java.util.Objects.requireNonNull;
+
+public class AddContactToTeamCommand extends Command {
+
+    public static final String COMMAND_WORD = "add-to-team";
```

Comment:
Since this is a team command, you can omit team from the command word. `add` is already used by `AddTeamCommand` so I recommend `add-contact`. An example command would be `team 1 add-contact 1`